### ISSUE TITLE
reusePaths removes object

### ISSUE THREAD

**Issue by JoKalliauer** [svg/svgo#1737]
## Input 
![Input](https://user-images.githubusercontent.com/15687581/211881812-00d1a4db-7a89-4595-8b28-635e71cf443d.svg)

## Output
![Output](https://user-images.githubusercontent.com/15687581/211881856-4a3aac86-c110-4ce4-8c32-c44192bb7b87.svg)

## Version
https://jakearchibald.github.io/svgomg/ (v3.0.0)

## Copyleft

source: https://commons.wikimedia.org/wiki/File:F16_Flight_Icon_blue.svg

author:
[File:F16 drawing.svg](https://commons.wikimedia.org/wiki/File:F16_drawing.svg): U.S. Government
    [File:P blank.svg](https://commons.wikimedia.org/wiki/File:P_blank.svg): [Chris-martin](https://commons.wikimedia.org/wiki/User:Chris-martin)
    abgeleitetes Werk [Sebastian Wallroth](https://commons.wikimedia.org/wiki/User:Sebastian_Wallroth)

license: Public-Domain

**Comment by SethFalco** [comment]
I've made a minimal example that allows us to reproduce the issue. I'll actually look into fixing it later, though. I've already made a PR to resolve the multiple `defs` issue in the optimized version, but that wasn't the problem here.

* https://github.com/svg/svgo/pull/1785

**`original.svg`**
```svg
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.0" viewBox="0 0 400 360">
  <defs>
    <path id="a" d="M51.94 428.2c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.02 5.28-29.47 10.43-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z"/>
    <path id="b" d="M51.94 428.2c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.02 5.28-29.47 10.43-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z"/>
    <clipPath id="c">
      <use xlink:href="#b" width="100%" height="100%" overflow="visible"/>
    </clipPath>
  </defs>
  <g transform="matrix(.491 0 0 .491 10.63 63.15)">
    <use xlink:href="#b" width="100%" height="100%" fill="#fff" fill-rule="evenodd" clip-rule="evenodd" overflow="visible"/>
    <path fill="none" stroke="#c8cacc" stroke-miterlimit="3.86" stroke-width="66.34" d="M48.33 412.36c14.5-32.39 36.89-59.5 64.39-81.96 13.75-11.23 65.03-24.09 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.09 28.32-2.2 1.9-38.65 17.58-43.77 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#c1c2c4" stroke-miterlimit="3.86" stroke-width="62.85" d="M48.52 413.19c14.5-32.39 36.89-59.5 64.39-81.96 13.75-11.23 65.03-24.09 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.09 28.32-2.2 1.9-38.65 17.59-43.77 19.51-14.01 5.28-29.47 10.44-44.3 12.72-3.2.49-14.99 3.84-16.61-.46z" clip-path="url(#c)"/>
    <path fill="none" stroke="#babcbe" stroke-miterlimit="3.86" stroke-width="59.36" d="M48.7 414.03c14.5-32.39 36.9-59.5 64.4-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.09 28.31-2.2 1.9-38.65 17.6-43.77 19.52-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#b2b4b7" stroke-miterlimit="3.86" stroke-width="55.87" d="M48.9 414.86c14.5-32.39 36.89-59.5 64.39-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.09 28.32-2.2 1.9-38.66 17.59-43.77 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#abadb0" stroke-miterlimit="3.86" stroke-width="52.38" d="M49.09 415.7c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.31-2.18 1.9-38.64 17.6-43.76 19.52-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#a4a6a9" stroke-miterlimit="3.86" stroke-width="48.89" d="M49.28 416.53c14.5-32.39 36.88-59.5 64.39-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.31-2.18 1.9-38.65 17.6-43.76 19.52-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#9d9fa2" stroke-miterlimit="3.86" stroke-width="45.4" d="M49.47 417.36c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.64 17.59-43.76 19.51-14.02 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#96989b" stroke-miterlimit="3.86" stroke-width="41.91" d="M49.66 418.2c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.64 17.58-43.76 19.51-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#909295" stroke-miterlimit="3.86" stroke-width="38.42" d="M49.85 419.03c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.65 17.59-43.76 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#898b8e" stroke-miterlimit="3.86" stroke-width="34.93" d="M50.04 419.86c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.65 17.59-43.76 19.51-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#828487" stroke-miterlimit="3.86" stroke-width="31.44" d="M50.23 420.7c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.02 5.28-29.47 10.44-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#7c7e80" stroke-miterlimit="3.86" stroke-width="27.95" d="M50.42 421.53c14.5-32.39 36.88-59.5 64.39-81.96 13.75-11.23 65.03-24.09 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.65 17.58-43.76 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#757779" stroke-miterlimit="3.86" stroke-width="24.46" d="M50.6 422.37c14.5-32.39 36.9-59.5 64.4-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#6f7073" stroke-miterlimit="3.86" stroke-width="20.96" d="M50.8 423.2c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.59-43.76 19.51-14.02 5.28-29.47 10.44-44.31 12.72-3.19.48-14.98 3.84-16.6-.46z" clip-path="url(#c)"/>
    <path fill="none" stroke="#68696c" stroke-miterlimit="3.86" stroke-width="17.48" d="M50.99 424.04c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.1 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.31-2.19 1.9-38.65 17.6-43.76 19.52-14.02 5.28-29.47 10.43-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#616264" stroke-miterlimit="3.86" stroke-width="13.98" d="M51.18 424.87c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.1 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.02 5.28-29.47 10.44-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#595a5c" stroke-miterlimit="3.86" stroke-width="10.49" d="M51.37 425.7c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#515254" stroke-miterlimit="3.86" stroke-width="7" d="M51.55 426.54c14.5-32.39 36.9-59.5 64.4-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51C98.45 419.57 83 424.71 68.16 427c-3.2.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#49494b" stroke-miterlimit="3.86" stroke-width="3.51" d="M51.75 427.37c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.1 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.65 17.58-43.76 19.51-14.01 5.28-29.47 10.43-44.31 12.71-3.2.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
  </g>
</svg>
```

**`svgo.config.js`**
```
module.exports = {
  plugins: [
    'reusePaths'
  ]
};
```

**`optimized.svg`**
```svg
<svg xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink" version="1.0" viewBox="0 0 400 360">
  <defs>
    <path id="a" d="M51.94 428.2c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.02 5.28-29.47 10.43-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z"/>
  </defs>
  <defs>
    <use xlink:href="#a"/>
    <use id="b" xlink:href="#a"/>
    <clipPath id="c">
      <use xlink:href="#b" width="100%" height="100%" overflow="visible"/>
    </clipPath>
  </defs>
  <g transform="matrix(.491 0 0 .491 10.63 63.15)">
    <use xlink:href="#b" width="100%" height="100%" fill="#fff" fill-rule="evenodd" clip-rule="evenodd" overflow="visible"/>
    <path fill="none" stroke="#c8cacc" stroke-miterlimit="3.86" stroke-width="66.34" d="M48.33 412.36c14.5-32.39 36.89-59.5 64.39-81.96 13.75-11.23 65.03-24.09 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.09 28.32-2.2 1.9-38.65 17.58-43.77 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#c1c2c4" stroke-miterlimit="3.86" stroke-width="62.85" d="M48.52 413.19c14.5-32.39 36.89-59.5 64.39-81.96 13.75-11.23 65.03-24.09 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.09 28.32-2.2 1.9-38.65 17.59-43.77 19.51-14.01 5.28-29.47 10.44-44.3 12.72-3.2.49-14.99 3.84-16.61-.46z" clip-path="url(#c)"/>
    <path fill="none" stroke="#babcbe" stroke-miterlimit="3.86" stroke-width="59.36" d="M48.7 414.03c14.5-32.39 36.9-59.5 64.4-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.09 28.31-2.2 1.9-38.65 17.6-43.77 19.52-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#b2b4b7" stroke-miterlimit="3.86" stroke-width="55.87" d="M48.9 414.86c14.5-32.39 36.89-59.5 64.39-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.09 28.32-2.2 1.9-38.66 17.59-43.77 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#abadb0" stroke-miterlimit="3.86" stroke-width="52.38" d="M49.09 415.7c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.31-2.18 1.9-38.64 17.6-43.76 19.52-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#a4a6a9" stroke-miterlimit="3.86" stroke-width="48.89" d="M49.28 416.53c14.5-32.39 36.88-59.5 64.39-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.31-2.18 1.9-38.65 17.6-43.76 19.52-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#9d9fa2" stroke-miterlimit="3.86" stroke-width="45.4" d="M49.47 417.36c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.64 17.59-43.76 19.51-14.02 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#96989b" stroke-miterlimit="3.86" stroke-width="41.91" d="M49.66 418.2c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.64 17.58-43.76 19.51-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#909295" stroke-miterlimit="3.86" stroke-width="38.42" d="M49.85 419.03c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.65 17.59-43.76 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#898b8e" stroke-miterlimit="3.86" stroke-width="34.93" d="M50.04 419.86c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.65 17.59-43.76 19.51-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#828487" stroke-miterlimit="3.86" stroke-width="31.44" d="M50.23 420.7c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.02 5.28-29.47 10.44-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#7c7e80" stroke-miterlimit="3.86" stroke-width="27.95" d="M50.42 421.53c14.5-32.39 36.88-59.5 64.39-81.96 13.75-11.23 65.03-24.09 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.65 17.58-43.76 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#757779" stroke-miterlimit="3.86" stroke-width="24.46" d="M50.6 422.37c14.5-32.39 36.9-59.5 64.4-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.01 5.28-29.47 10.43-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#6f7073" stroke-miterlimit="3.86" stroke-width="20.96" d="M50.8 423.2c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.59-43.76 19.51-14.02 5.28-29.47 10.44-44.31 12.72-3.19.48-14.98 3.84-16.6-.46z" clip-path="url(#c)"/>
    <path fill="none" stroke="#68696c" stroke-miterlimit="3.86" stroke-width="17.48" d="M50.99 424.04c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.1 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.31-2.19 1.9-38.65 17.6-43.76 19.52-14.02 5.28-29.47 10.43-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#616264" stroke-miterlimit="3.86" stroke-width="13.98" d="M51.18 424.87c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.1 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.02 5.28-29.47 10.44-44.31 12.71-3.19.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#595a5c" stroke-miterlimit="3.86" stroke-width="10.49" d="M51.37 425.7c14.5-32.39 36.89-59.5 64.38-81.96 13.76-11.23 65.04-24.09 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.22 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51-14.01 5.28-29.47 10.44-44.3 12.71-3.2.5-14.99 3.85-16.61-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#515254" stroke-miterlimit="3.86" stroke-width="7" d="M51.55 426.54c14.5-32.39 36.9-59.5 64.4-81.96 13.75-11.23 65.03-24.1 73.85-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.19 1.9-38.65 17.58-43.76 19.51C98.45 419.57 83 424.71 68.16 427c-3.2.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
    <path fill="none" stroke="#49494b" stroke-miterlimit="3.86" stroke-width="3.51" d="M51.75 427.37c14.5-32.39 36.88-59.5 64.38-81.96 13.76-11.23 65.04-24.1 73.86-16.58 9.45 8.06 13.45 26.18 5.53 38.45-1.23 1.9-37.38 26.83-39.1 28.32-2.18 1.9-38.65 17.58-43.76 19.51-14.01 5.28-29.47 10.43-44.31 12.71-3.2.5-14.98 3.85-16.6-.45z" clip-path="url(#c)"/>
  </g>
</svg>
```

**Comment by JoKalliauer** [comment]
confrim fix in 3.0.3
